client/src/App.js

```diff
@@ -1,19 +1,22 @@
-import React, { Component } from "react";
-import { BrowserRouter, Route } from "react-router-dom";
-import "tachyons/css/tachyons.css";
+import React, { Component } from 'react';
+import { BrowserRouter, Route } from 'react-router-dom';
+import 'tachyons/css/tachyons.css';
+import Landing from './components/Landing';
+import Navbar from './components/Navbar';
+import CategoryForm from './containers/CategoryForm';
 
-import Landing from "./components/Landing";
-import Navbar from "./components/Navbar";
-
-const App = () => {
-  return (
-    <BrowserRouter>
-      <div>
-        <Navbar />
-        <Route path="/" component={Landing} />
-      </div>
-    </BrowserRouter>
-  );
-};
+class App extends Component {
```

Comment:
why does this need to be a class component again ?
youre not using any of the `Component` class methods or a state